R/pkg/R/catalog.R

```diff
@@ -118,6 +118,7 @@ createExternalTable <- function(tableName, path = NULL, source = NULL, schema =
 #'
 #' @param tableName the qualified or unqualified name that designates a table. If no database
 #'                  identifier is provided, it refers to a table in the current database.
+#'                  Since 3.4.0 it is allowed to be qualified with catalog name.
```

Comment:
it sounds slightly awkward (to me). I would write like, "The table name can be fully qualified with catalog name since 3.4.0."

Reply:
will update soon